time stamp
float32[] sensor_ranges
float32[] points_x
float32[] points_y
float32 robot_x
float32 robot_y
float32 robot_theta

    
   

